### PR TITLE
[FIX] stock: unreserve when changing location on import

### PR DESCRIPTION
**Pull request by nouraellm**
### Steps to reproduce:

- Install stock and enable locations
- Enable import on quants tree view
- Create a transfer and reserve [E-COM07] product
- Go to quants or current stock of the location
- Export [E-COM07] then change the location on the exported file
- Import the file you edited

### Current behavior:

- Unreserve issue is triggered because [E-COM07] quants are reserved.

### Current behavior:

- Only allowed fields should be imported, and that doesn't include the `location_id`.

- Task id: 3007499

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by nouraellm** [comment]
@ticodoo here you go!

**Comment by ticodoo** [comment]
@nouraellm ah dang sorry I forgot to tell you that since it's adding a new user facing string in a stable version, you also have to add the new UserError into the .pot file for translation purposes. Let me know if you need guidance for how to do that.

**Comment by nouraellm** [comment]
@ticodoo my bad I always forget about translations 😅 should be good now

File: addons/stock/i18n/stock.pot [hunk 1]
```diff
@@ -1229,6 +1229,12 @@ msgid ""
 "legislation of the country in which the client is located."
 msgstr ""
 
+#. module: stock
+#: code:addons/stock/models/stock_quant.py:0
+#, python-format
+msgid "Changing %s is restricted, you can't do this operation."
+msgstr ""
+
```
[the remark this answers is not in the document]
Comment (nouraellm) [reply addons/stock/i18n/stock.pot]: @ticodoo I was not aware of this, thanks!

**Comment by ticodoo** [comment]
@robodoo r+

**Pull request closed without merging**